CefSharp.Wpf.Example/Views/BrowserTabView.xaml

```diff
@@ -168,12 +168,15 @@
                     </Rectangle.Fill>
                 </Rectangle>
 
+                    <ContentControl Content="{Binding WebBrowser}"
+                                Opacity="{Binding ElementName=opacitySlider, Path=Value}" >
+                    <!--
                 <cefSharp:ChromiumWebBrowser x:Name="browser"
```

Comment:
Whilst your way of creating the `ChromiumWebBrowser` instance will obviously work, I think the extra level of indirection is not great for an example application.